feg/protos/s6a_proxy.proto

```diff
@@ -107,6 +107,8 @@ message AuthenticationInformationAnswer {
         bytes xres = 2;
         bytes autn = 3;
         bytes kasme = 4;
+        bytes ck = 5;
+        bytes ik = 6;
```

Comment:
do not make any changes to EUTRANVector or other vectors. See my other comment about 5G Authentication Vector.

Reply:
removed/reverted above changes are done for 5G .